SU-SWS/stanford_profile
# D8CORE-971: fix home and no 2nd menu

# READY FOR REVIEW

# Summary

This fixes the overly broad targeting of
https://github.com/SU-SWS/stanford_profile/pull/65

_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-971_

# Needed Review By (Date)
- Soon

# Urgency
- Fixes broken stuff

# Steps to Test

- Checkout this branch
- Verify that the spacing on the homepage looks correct (no weird padding)
- Verify that the spacing on a page with no secondary nav doesn't have the left padding
- Verify that the spacing on a page with secondary nav has the expected left padding

# Affected Projects or Products 
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# QA


## Cross-Browser testing:
What browsers have been tested?
- Chrome Desktop - Yes
- Chrome Android
- FireFox Desktop - Yes
- FireFox Android
- OSX Safari
- iOS Safari
- Edge
- IE 11

## Unit Testing


# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-971

## Related PRs
https://github.com/SU-SWS/stanford_profile/pull/65

## More Information

## Folks to notify
`@sherakama` 
`@kerri-augenstein`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
